tests/unit/amo/components/TestInstallButtonWrapper.js

```diff
@@ -83,92 +140,67 @@ describe(__filename, () => {
   };
 
   it(`calls getClientCompatibility with the add-on's current version if no version is supplied`, () => {
-    const addon = fakeAddon;
-
-    _loadVersions({ slug: addon.slug, versions: [addon.current_version] });
-
     const clientApp = CLIENT_APP_FIREFOX;
-    const _getClientCompatibility = jest.fn().mockReturnValue({
-      compatible: true,
-    });
 
     _dispatchClientMetadata({
       clientApp,
     });
 
-    render({
-      _getClientCompatibility,
-      addon: createInternalAddonWithLang(addon),
-    });
+    renderWithCurrentVersion();
 
     expect(_getClientCompatibility).toHaveBeenCalledWith({
-      addon: createInternalAddonWithLang(addon),
+      addon: createInternalAddonWithLang(fakeAddon),
       clientApp,
-      currentVersion: createInternalVersionWithLang(addon.current_version),
+      currentVersion: createInternalVersionWithLang(fakeAddon.current_version),
       userAgentInfo: store.getState().api.userAgentInfo,
     });
   });
 
   it(`calls getClientCompatibility with a specific version if supplied`, () => {
-    const slug = 'some-slug';
-    const addon = { ...fakeAddon, slug };
     const version = { ...fakeVersion, id: fakeVersion.id + 1 };
 
-    _loadVersions({ slug, versions: [version] });
+    _loadVersions({ slug: fakeAddon.slug, versions: [version] });
 
     const clientApp = CLIENT_APP_FIREFOX;
-    const _getClientCompatibility = jest.fn().mockReturnValue({
-      compatible: true,
-    });
 
     _dispatchClientMetadata({
       clientApp,
     });
 
     render({
-      _getClientCompatibility,
-      addon: createInternalAddonWithLang(addon),
       version: createInternalVersionWithLang(version),
     });
 
     expect(_getClientCompatibility).toHaveBeenCalledWith({
-      addon: createInternalAddonWithLang(addon),
+      addon: createInternalAddonWithLang(fakeAddon),
       clientApp,
       currentVersion: createInternalVersionWithLang(version),
       userAgentInfo: store.getState().api.userAgentInfo,
     });
   });
 
   it('does not call getClientCompatibility when the browser is not Firefox', () => {
-    const addon = fakeAddon;
-
     const clientApp = CLIENT_APP_FIREFOX;
-    const _getClientCompatibility = jest.fn();
 
     _dispatchClientMetadata({
       clientApp,
       userAgent: userAgentsByPlatform.mac.chrome41,
     });
 
-    render({
-      _getClientCompatibility,
-      addon: createInternalAddonWithLang(addon),
-    });
+    render();
```

Comment:
This is a good example of a test where it's fine to do away with `addon`, and rely on the default, because the test doesn't assert about anything involving the `addon`.